### PR TITLE
Core circular dependencies refactor

### PR DESCRIPTION
**Pull request by paulSanchezMarmelab**
Fixed some blocking circular dependencies in core module.

### CODE REVIEW

File: mobileapp/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  "version": "1.0.0",
+  "main": "./AppEntry.js",
+  "scripts": {
+    "start": "expo start",
+    "android": "expo start --android",
+    "ios": "expo start --ios",
+    "web": "expo start --web"
+  },
+  "dependencies": {
+    "@kamon/core": "1.0.0",
```
Comment (Nitix) [inline mobileapp/package.json:12]: It should be `"@kamon/core": "workspace:^",`

File: webapp/package.json [hunk 2]
```diff
@@ -24,7 +24,7 @@
     "db:migrate": "yarn run typeorm migration:run -d src/ormconfig.ts"
   },
   "dependencies": {
-    "@kamon/core": "workspace:^",
+    "@kamon/core": "1.0.0",
```
Comment (Nitix) [inline webapp/package.json:27]: `"@kamon/core": "workspace:^",` was good.

File: mobileapp/package.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    "expo": "~48.0.15",
+    "expo-status-bar": "~1.4.4",
+    "react": "18.2.0",
+    "react-native": "0.71.7"
```
Comment (Nitix) [inline mobileapp/package.json:16]: Nice deps are up to date :wink:

**Review by Nitix — COMMENTED** [review]
Quick review. Guilbill is the real reviewer.

**Comment by fzaninotto** [comment]
Can you rebase?

**Comment by fzaninotto** [comment]
@paulSanchezMarmelab bump, please rebase.

**Comment by paulSanchezMarmelab** [comment]
It's been merged with main.

**Pull request merged**